src/ui-presets/playback.js

```diff
@@ -71,6 +71,7 @@ class PlaybackUI extends Component {
             <VideoArea />
             <GuiArea>
               <Fragment>
+                <UnmuteIndication />
```

Comment:
why ?

Reply:
<img width="657" alt="Screen Shot 2022-09-11 at 12 50 55" src="https://user-images.githubusercontent.com/62672270/189523743-102a34a3-41f1-4d72-a74c-77c5ab7a9267.png">

Reply:
while top bar contains icons it falls bellow